meta_model_building/get_opt_params.py

```diff
@@ -1,40 +1,67 @@
+import numpy as np
 import os
 import pickle
 from tpot import TPOTRegressor
 from sklearn.datasets import load_boston
 from sklearn.model_selection import train_test_split
+import sys
 import time
 
+if len(sys.argv) > 1:
+    input_file_name = sys.argv[1]
+    if sys.argv[1] == 'boston':
+        housing = load_boston()
+        data, target = housing.data, housing.target
+    else:
+        try:
+            file_data = np.genfromtxt(input_file_name, delimiter=',')
+            data, target = file_data[:, :-1], file_data[:, -1:] 
+            target = np.ravel(target)
+        except:
+            print("Failed to read data file", input_file_name, "as CSV.  Aborting.")
+            quit()
+else:
+    input_file_name = 'boston'
+    housing = load_boston()
+    data, target = housing.data, housing.target
+
+seed = {
+    'split_seed': 42,
+    'tpot_seed': 42}
+
 run_param = {
-	'random_state_train': 42,
-	'random_state_tpot': 42,
-	'export_python_code_filename': 'tpot_boston_pipeline.py',
-	'pickle_file_name': 'boston_pipeline_evaluated_models.pickle',
-	'train_size': 0.75,
-	'test_size': 0.25,
-	'generations': 5,
-	'population_size': 20,
-	'verbosity': 2,
-	'time': time.time()
-}
+    'train_size': 0.75,
+    'test_size': 0.25,
+    'generations': 5,
+    'population_size': 20,
+    'verbosity': 3}
 
 housing = load_boston()
-X_train, X_test, y_train, y_test = train_test_split(housing.data, housing.target,
+x_train, x_test, y_train, y_test = train_test_split(data, target,
                                                     train_size=run_param['train_size'], 
                                                     test_size=run_param['test_size'],
-						    random_state = run_param['random_state_train'])
+                                                    random_state = seed['split_seed'])
+
+tpot = TPOTRegressor(generations=run_param['generations'], population_size=run_param['population_size'],
+                    verbosity=run_param['verbosity'], random_state = seed['tpot_seed'])
 
-tpot = TPOTRegressor(generations=run_param['generations'], population_size=run_param['population_size'], 
-					 verbosity=run_param['verbosity'], random_state = run_param['random_state_tpot'])
+tpot.fit(x_train, y_train)
+print(tpot.score(x_test, y_test))
 
-tpot.fit(X_train, y_train)
-print(tpot.score(X_test, y_test))
-tpot.export(run_param['export_python_code_filename'])
+time = time.time()
```

Comment:
This is a little... iffy. You're overwriting the handle to the library you imported. Rename the variable to something like `start_time` which is probably a better label anyway.

Reply:
Fixed (in dev_get-boston-data_20180924).